flaml/integrations/oai/completion.py

```diff
@@ -172,25 +195,41 @@ def eval(cls, config: dict, prune=True, eval_only=False):
         Args:
             config (dict): Hyperparameter setting for the openai api call.
             prune (bool, optional): Whether to enable pruning. Defaults to True.
-            eval_only (bool, optional): Whether to evaluate only. Defaults to False.
+            eval_only (bool, optional): Whether to evaluate only (ignore the inference budget and no timeout).
+              Defaults to False.
 
         Returns:
             dict: Evaluation results.
         """
         cost = 0
         data = cls.data
+        model = config["model"]
         data_length = len(data)
-        target_n_tokens = (
-            1000 * cls.inference_budget / cls.price1K[config["model"]]
-            if cls.inference_budget and cls.price1K.get(config["model"])
+        target_n_tokens = getattr(cls, "inference_budget", None) and (
+            1000 * cls.inference_budget / cls.price1K[model]
```

Comment:
```suggestion
            1000 * cls.inference_budget / cls.price1K.get(model)
```

Reply:
This will not work. If model is not in the dict, `get()` will return None.